Private Registry Modules via ghapps (#138)
hashicorp/terraform-docs-common: website/docs/cloud-docs/vcs/github-app.mdx

## Feature Limitations

> The Terraform Cloud GitHub App is designed for creating workspaces and private registry modules from the Terraform Cloud UI. It currently does not support:

Reviewer: ```suggestion
You can use the Terraform Cloud GitHub App to create workspaces and private registry modules from the Terraform Cloud UI. It currently does not support:
```

Suggest changing "is designed for creating" to "You can use...to create" to avoid a passive voice style issue.

Author: Thank you for the suggestion, Matthew!
